nix/docker.nix

```diff
@@ -0,0 +1,141 @@
+############################################################################
+# Docker image builder
+#
+# To build and load into the Docker engine:
+#
+#   docker load -i $(nix-build -A dockerImage.explorerApi --no-out-link)
+#   docker load -i $(nix-build -A dockerImage.txSubmit --no-out-link)
+#
+# To launch with provided mainnet configuration and persist state in a docker
+# volume run:
+#
+#   docker run -v /data -e NETWORK=mainnet inputoutput/cardano-explorer-api:<TAG>
```

Comment:
```suggestion
#   docker run NETWORK=mainnet inputoutput/cardano-explorer-api:<TAG>
```

Reply:
Cardano Explorer API isn't concerned about the network, it just connects to the DB